server/finatra-server/src/main/scala/tapir/server/finatra/OutputToFinatraResponse.scala

```diff
@@ -0,0 +1,144 @@
+package tapir.server.finatra
+import java.io.{File, InputStream}
+import java.nio.ByteBuffer
+
+import com.twitter.finagle.http.Status
+import com.twitter.io.{Buf, InputStreamReader, Reader}
+import org.apache.http.entity.ContentType
+import org.apache.http.entity.mime.content._
+import org.apache.http.entity.mime.{FormBodyPart, FormBodyPartBuilder, MultipartEntityBuilder}
+import tapir.internal.{ParamsToSeq, _}
+import tapir.model.Part
+import tapir.{
+  ByteArrayValueType,
+  ByteBufferValueType,
+  CodecMeta,
+  EndpointIO,
+  EndpointOutput,
+  FileValueType,
+  InputStreamValueType,
+  MediaType,
+  MultipartValueType,
+  RawPart,
+  StreamingEndpointIO,
+  StringValueType
+}
+
+object OutputToFinatraResponse {
```

Comment:
take a look at `EncodeOutputs` - some of this logic has now been extracted to a common object (much like `DecodeInputs`)

Reply:
Thanks, I like the new `EncodeOutputs` !